### PR TITLE
update: add 1.6.10-RC EAP details

### PR DESCRIPTION
**Pull request by koshachy**

### CODE REVIEW

File: docs/topics/eap.md [hunk 1]
```diff
@@ -26,4 +26,33 @@ check [our instructions on how to configure your build to support this version](
...
+    </tr>
+    <tr>
+        <td><strong>1.6.10-RC</strong>
+            <p>Released: <strong>December 7, 2021</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.10-RC" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+             <ul>
+                <li>Release candidate for a bug fix release for Kotlin 1.6.0.</li>
+            </ul>
```
Comment (p7nov) [inline docs/topics/eap.md:43]: I wouldn't use the list for this single item.

Comment (p7nov) [reply docs/topics/eap.md:43]: Also, there are two "for"s in a short sentence. Maybe "Release candidate for a bug fix release Kotlin 1.6.10."?

**Review by p7nov — APPROVED** [review]
Minor improvement suggestions

File: docs/topics/eap.md (hunk 1 above)
Comment (koshachy) [reply docs/topics/eap.md:43]: Thanks, I like this

File: docs/topics/eap.md [hunk 2]
```diff
@@ -26,4 +26,33 @@ check [our instructions on how to configure your build to support this version](
...
+        <th>Recommended kotlinx library versions</th>
+    </tr>
+    <tr>
+        <td><strong>1.6.10-RC</strong>
+            <p>Released: <strong>December 7, 2021</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.10-RC" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+             <ul>
+                <li>Release candidate for a bug fix release for Kotlin 1.6.0.</li>
```
Comment (koshachy) [inline docs/topics/eap.md:42]:
```suggestion
                <li>Release candidate for a bug fix release Kotlin 1.6.10.</li>
```

File: docs/topics/eap.md (hunk 1 above)
Comment (koshachy) [inline docs/topics/eap.md:43]:
```suggestion
        <td>
             <p>Release candidate for a bug fix release for Kotlin 1.6.0.</p>
```

**Review by woainikk — APPROVED** [review]

**Pull request merged**